components/component-stylesheets/button.css

```diff
@@ -60,16 +60,16 @@
 }
 
     /* icon buttons */
-    .btn-icon{
-        background: none;
-        border: none;
-        font-size: 1.8rem;
-        color: var(--mr-primary-color);
-    }
+.btn-icon{
+    background: none;
+    border: none;
+    font-size: 1.8rem;
+    color: var(--mr-primary-color);
+}
 
-    .btn-icon:hover{
-        color: var(--mr-dark-primary);
-    }
```

Comment:
What is Mr?

Reply:
it is not obvious in the first go

Reply:
its naming convention for variables declared for colors in a separate CSS file. I think will not trouble the user, all the variables declared for colors can be seen in the file, Please tell if there is any issue.